subsys/caf/modules/power_manager.c

```diff
@@ -132,6 +133,10 @@ static void system_off(void)
 		set_power_state(POWER_STATE_OFF);
 		LOG_WRN("System turned off");
 		LOG_PANIC();
+		/* Reset RESETREAS to avoid starting serial recovery if nobody
```

Comment:
If you don't mind: `s/Reset/Clear`. Not a blocker

Reply:
Done